go.mod

```diff
@@ -103,3 +103,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
Once everyone agrees with the implementation I will create a separate PR in order to fix the go.mod here.